charts/firefly/README.md

```diff
@@ -387,9 +383,9 @@ which provides Fabconnect alongside its Fabric peer nodes.
 
 Once you have a Fabconnect instance ready, FireFly then needs three pieces of configuration:
 
-* `config.organizationKey`: the name of the organization's Fabric identity which will be used for signing transactions
-* `config.fabconnectUrl`: the HTTP/S URL of the Fabconnect instance FireFly will use
-* `config.fabconnectSigner`:  the name of the organization's Fabric identity which will be used for signing transactions.
+- `config.organizationKey`: the name of the organization's Fabric identity which will be used for signing transactions
+- `config.fabconnectUrl`: the HTTP/S URL of the Fabconnect instance FireFly will use
+- `config.fabconnectSigner`: the name of the organization's Fabric identity which will be used for signing transactions.
```

Comment:
Have we tested fab connect?

Reply:
I personally haven't, worth noting that these configuration options would be outside the scope of what the tooling could generate so we'd need to stand up a custom stack to test this out, interested to know if we see this as a blocker for this PR or if this is future work.